log_drains/add.go

```diff
@@ -7,17 +7,40 @@ import (
 	"gopkg.in/errgo.v1"
 )
 
-func Add(app string, params scalingo.LogDrainAddParams) error {
+func Add(app string, opts ListAddonOpts, params scalingo.LogDrainAddParams) error {
```

Comment:
I wouldn't add a third argument here for `ListAddonOpts`. I think I prefer just a single argument for this function. It makes more sense. Don't you think?